build.gradle

```diff
@@ -437,7 +437,7 @@ if (project.hasProperty('mavenUser') && project.hasProperty('jcecertAlias')) {
         }
     }
 
-    task sign(overwrite: true) {
+    task sign {
```

Comment:
Any more context on this change?

Reply:
With gradle upgrade from 5 to 7, using 'overwrite' is invalid if the task doesn't already exist. If we don't make this change, gradle fails with the error:
```
> Could not create task ':sign'.
   > Unnecessarily replacing a task that does not exist is not supported.  Use create() or register() directly instead.  You attempted to replace a task named 'sign', but there is no existing task with that name.
```
My guess is it was not necessary for `sign` to overwrite in the first place.